File: src/main/java/com/qsoftware/bubbles/command/GameOverCommand.java
package com.qsoftware.bubbles.command;

import com.qsoftware.bubbles.common.command.Command;
import com.qsoftware.bubbles.common.command.CommandExecutor;
import com.qsoftware.bubbles.entity.player.PlayerEntity;

public class GameOverCommand implements CommandExecutor {
    @Override
    public boolean execute(Command command, PlayerEntity player, String[] args) {
        player.getGameType().triggerGameOver();
        return true;
    }
}

File: src/main/kotlin/qtech/bubbles/command/GameOverCommand.kt
package qtech.bubbles.command

import qtech.bubbles.common.command.CommandExecutor
import qtech.bubbles.entity.player.PlayerEntity

open class GameOverCommand : CommandExecutor {
    override fun execute(player: PlayerEntity, args: Array<String>): Boolean {
        player.gameMode.triggerGameOver()
        return true
    }
}